### PR TITLE
docs: update crud limitations for compoundindexer

### PR DESCRIPTION
**Pull request by CatStark**
### All Submissions:

* [x] Have you followed the [documentation style guide](https://github.com/jina-ai/docs/tree/master/page_templates/style_guide.md)? 

### New Feature Submissions:

1. [x] Have you added your new file to a table of content index? 
2. [x] Did you use one of the templates in the [page_template](https://github.com/jina-ai/docs/tree/master/page_templates) folder?
3. [x] Is your title in Title Case? 
4. [x] Are your section headers in sentence case?

### CODE REVIEW

**Review by rutujasurve94 — APPROVED** [review]

File: chapters/crud.md [hunk 1]
```diff
@@ -66,3 +66,16 @@ You can see this in the code listing in the beginning of this chapter.
    
 The update and delete operations use a masking underneath. This is done to maintain high performance overall. However, this means that old data will not be deleted, but will simply be masked as being deleted. Thus the size on disk (and in memory) of the indexer will grow over time if you perform update or delete operations. We recommend setting the `delete_on_dump` parameter of the `NumpyIndexer` to `True`. When the Flow is shut down, the data that has been marked as deleted will be permanently deleted before being saved to disk. By default the parameter is set to `False`, as setting it to `True` will make the shut down process slower.  
 
+
+6. **Compound Indexer**
+
+While using a `!CompoundIndexer`, it is not possible to return embeddings by virtue of its design. By definition, the [!CompoundIndexer](https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.CompoundIndexer.yml) has an `!ExcludeQL` for all the embeddings:
```
Comment (cristianmtr) [inline chapters/crud.md]: Not sure I understand what 'return embeddings' means

Comment (rutujasurve94) [reply chapters/crud.md]: @theUnkownName by `return` do we mean that Vector + KV indexer (Compound) won't persist the embedding?

Comment (cristianmtr) [reply chapters/crud.md]: I think I got it. You meant return to the next Driver in the Flow? Yes, that would be the case with the current Exclude

Comment (cristianmtr) [reply chapters/crud.md]: Maybe specify that in more detail

Comment (cristianmtr) [inline chapters/crud.md]:
```suggestion
While using a `!CompoundIndexer`, it is not possible to return embeddings to the next Driver in the Flow, by virtue of its design. By definition, the [!CompoundIndexer](https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.CompoundIndexer.yml) has an `!ExcludeQL` for all the embeddings:
```

Comment (CatStark) [reply chapters/crud.md]: It is what we discussed yesterday on the daily, that because of the ExcludeQL is not possible to keep the embeddings

Comment (CatStark) [reply chapters/crud.md]: Sounds better, I made the change, what you think now? @cristianmtr

**Review by cristianmtr — APPROVED** [review]
LGTM

**Pull request merged**